src/main/java/uk/ac/cam/cl/dtg/isaac/quiz/IsaacNumericValidator.java

```diff
@@ -208,12 +208,13 @@ public final QuestionValidationResponse validateQuestionResponse(
         } catch (NumberFormatException e) {
             log.debug("Validation failed for '" + answerFromUser.getValue() + " " + answerFromUser.getUnits() + "': "
                     + "cannot parse as number!");
-
-            String feedback = "Your answer is not in a format we recognise, please enter your answer as a decimal number.";
+            HashSet<String> responseTags = new HashSet<>(ImmutableList.of("nq_format"));
             if (answerFromUser.getValue().matches(INVALID_NEGATIVE_STANDARD_FORM)) {
-                feedback += "<br>When writing standard form, you must include a `^` between the 10 and the exponent.";
```

Comment:
(But you can leave this line out now, and just add the tag, I think that's fine).